### PR TITLE
More 2.0 SDK work

### PR DESCRIPTION
**Pull request by jaredpar**

### CODE REVIEW

File: build/Targets/Roslyn.Toolsets.Xunit.targets [hunk 1]
```diff
@@ -28,6 +24,7 @@
     <PropertyGroup>
       <_AlreadyHasAppConfig Condition="'%(None.Filename)%(None.Extension)' == 'app.config'">true</_AlreadyHasAppConfig>
       <_DefaultAppConfigFile Condition="'$(TargetFramework)' == 'net46'">$(MSBuildThisFileDirectory)..\Defaults\Net460\app.config</_DefaultAppConfigFile>
+      <_DefaultAppConfigFile Condition="'$(TargetFramework)' == 'net46'">$(MSBuildThisFileDirectory)..\Defaults\Net461\app.config</_DefaultAppConfigFile>
```
Comment (tmat) [inline build/Targets/Roslyn.Toolsets.Xunit.targets]: The condition is missing '1'

File: src/Tools/BuildBoss/ProjectCheckerUtil.cs [hunk 2]
```diff
@@ -62,6 +62,7 @@ public bool Check(TextWriter textWriter)
                 allGood &= CheckProjectReferences(textWriter);
                 allGood &= CheckPackageReferences(textWriter);
                 allGood &= CheckDeploymentSettings(textWriter);
+                // allGood &= CheckTargetFrameworks(textWriter);
```
Comment (tmat) [inline src/Tools/BuildBoss/ProjectCheckerUtil.cs]: Commented out?

Comment (jaredpar) [inline src/Tools/BuildBoss/ProjectCheckerUtil.cs]: @tmat this PR is still "for personal review". I wanted to verify some stuff with Jenkins and commented out this block temporarily to not get blocked by it.

**Comment by jaredpar** [comment]
retest ubuntu_14_debug_prtest please

**Pull request merged**